build.sbt

```diff
@@ -64,3 +64,13 @@ lazy val microservice = Project(appName, file("."))
     // Make sure you only exclude warnings for the project directories, i.e. make builds reproducible
     scalacOptions += s"-P:silencer:sourceRoots=${baseDirectory.value.getCanonicalPath}"
   )
+
+dependencyUpdatesFilter -= moduleFilter(organization = "com.typesafe.play")
+dependencyUpdatesFilter -= moduleFilter(name = "simple-reactivemongo")
+dependencyUpdatesFilter -= moduleFilter(name = "reactivemongo-test")
+dependencyUpdatesFilter -= moduleFilter(name = "domain")
+dependencyUpdatesFilter -= moduleFilter(name = "scala-library")
+dependencyUpdatesFilter -= moduleFilter(name = "flexmark-all")
+dependencyUpdatesFilter -= moduleFilter(name = "scalatestplus-play")
+dependencyUpdatesFilter -= moduleFilter(name = "scalatestplus-scalacheck")
+dependencyUpdatesFailBuild := true
```

Comment:
what does this stuff do?

Reply:
ignores the libraries that require updates that need Play 2.7 or higher, when most of our apps are on Play 2.6, "dependencyUpdatesFailBuild := true" will fail the build if any other libraries are out of date